packages/turbo/ios/RNVisitableView.swift

```diff
@@ -10,22 +10,53 @@ import UIKit
 
 class RNVisitableView: UIView, SessionSubscriber {
   var id: UUID = UUID()
+  @objc var sessionHandle: NSString? = nil {
+    didSet {
+      validateSessionProperty(propertyName: "sessionHandle", oldValue: oldValue, newValue: sessionHandle)
+    }
+  }
+  @objc var applicationNameForUserAgent: NSString? = nil {
+    didSet {
+      validateSessionProperty(propertyName: "applicationNameForUserAgent", oldValue: oldValue, newValue: applicationNameForUserAgent)
+    }
+  }
   @objc var url: NSString = "" {
     didSet {
       if (oldValue != "") {
```

Comment:
Might be able to drop this conditional, see https://github.com/software-mansion-labs/react-native-turbo-demo/pull/57/files#r1405894436